src/conversion_helper.js

```diff
@@ -73,6 +73,21 @@ function pressureMap(pa) {
   }
 }
 
+function velocityMap(ms) {
```

Comment:
Can `ms` be changed to `mPerS` or `metersPerSecond` or something? I first though `ms` stood for milliseconds, and was momentarily confused